src/opm/parser/eclipse/share/keywords/000_Eclipse100/O/OVERBURD

```diff
@@ -0,0 +1,7 @@
+{"name" : "OVERBURD" , "sections" : ["PROPS"], "num_tables" : {"keyword" : "TABDIMS" , "item" : "NTROCC"},
+	"size"     :  "UNKNOWN" ,
```

Comment:
You are aware that a hand-written application - on papyrus - is required to get keywords with size type `UNKNOWN` accepted?

Reply:
other suggestions are welcome provided that they do not require several man-months of work. the OVERBURD and ROCK2D -- and to a slightly lesser degree -- ROCKCOMP keywords are funny beasts. You should probably look them up yourself...

Reply:
OK; I think this should work:

1. Remove the `num_tables:` and `size:` keywords.
2. Add `"size" : {"keyword" : "ROCKCOMP", "item" : "NTROCC"}`

??

Reply:
maybe we confused this a bit: is `size` the number of records or the number of items? if the number of items is arbitrary, I guess that we can chicken out here as you proposed: e.g., for ROCKCOMP the problem is that the number of columns depends on the presence of the RKTRMDIR keyword...

Reply:
> is size the number of records or the number of items

The number of records.

> if the number of items is arbitrary,

The number of items is arbitrary - in your case - since you are using the *item-specific* `"size_type" : "ALL"`

Reply:
> You are aware that a hand-written application - on papyrus - is required to get keywords with size type UNKNOWN accepted?

 I tried different approaches, but gave up and used UNKNOWN. But maybe I just confused num_tables and size. I will try your solution.

Reply:
`"size" : {"keyword" : "ROCKCOMP", "item" : "NTROCC"}`

gives std::invalied_argument for ROCK2D. 

I have NTROCC tables where each of the table consists of an arbitary number of tables with oone entry for first column and arbitary (should be the same as in ROCKWNOD ) number of entries for the second column. Can I spesify nested tables?

Reply:
OK - the `ROCK2D` keyword looks like a perfectly standard keyword, i.e. it is terminated with a slash in the normal way. This should be a valid json configuration:
```json
{"name" : "ROCK2D" , "sections" : ["PROPS"], 
        "items" : [
            {"name":"PRESSURE", "value_type" : "DOUBLE", "dimension":"Pressure" },
            {"name":"PVMULT", "value_type":"DOUBLE", "size_type" : "ALL" , "dimension" : "1"}
         ]
}
```

Reply:
The coupling between the number of items in the `ROCK2D` tables and the `ROCKWNOD` tables can not be specified in the json - that must be checked in the application code.

Reply:
> This should be a valid json configuration:

with your next suggestion I get

`unknown location(0): fatal error in "Rock2d": std::invalid_argument: PARSE_RANDOM_TEXT: String '0.7      0.04' not formatted/recognized as valid keyword at: "":25`

i.e. it fails when it tried to read the second table. 

You can check RockTableTests.cpp for details on the input.

Reply:
Sorry - I misunderstood the documentation; you will need the `num_tables` setting. I'll call you.